docs/deprecated.md

```diff
@@ -1,8 +1,11 @@
 <!--[metadata]>
 +++
-title = "registry deprecation"
+title = "Deprecated Features"
```

Comment:
👍  - this is the master version, when I cherry-pick I'll need to resolve the merge-conflict